pandas/io/formats/style.py

```diff
@@ -63,6 +63,8 @@ class Styler(object):
         a unique identifier to avoid CSS collisons; generated automatically
     caption: str, default None
         caption to attach to the table
+    disabled_mathjax: bool, default False
```

Comment:
rather than an option prefer this to be an options as @TomAugspurger discussed in the orgiinal issue, and IIRC this should only be handled in the to_html, and NOT in the styler